### PR TITLE
HashGenerator output on base64 or hex

### PR DESCRIPTION
**Pull request by andrescirulo**
Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Internationalization and localization
- [ ] Other (please describe):

## What is the current behavior?

The output of a hash generation is always Hex. Sometimes is useful for the output to be in base64.

Issue Number: #239 

## What is the new behavior?

- An option to select the Output Type between Hex and Base64 has been added

### CODE REVIEW

File: src/dev/impl/DevToys/ViewModels/Tools/Generators/HashGenerator/HashGeneratorToolViewModel.cs [hunk 1]
```diff
@@ -161,7 +185,15 @@ private async Task<string> CalculateHashAsync(string alrogithmName, string text)
                 IBuffer buffer = CryptographicBuffer.ConvertStringToBinary(text, BinaryStringEncoding.Utf8);
                 buffer = algorithmProvider.HashData(buffer);
 
-                string? hash = CryptographicBuffer.EncodeToHexString(buffer);
```
Comment (btiteux) [inline src/dev/impl/DevToys/ViewModels/Tools/Generators/HashGenerator/HashGeneratorToolViewModel.cs:164]: Hello, 
I see that you compare two different values, can you use a String.Equals to do that?
And also please avoid using string directly here and define an internal for example, you can see and example in the "JsonYamlToolViewModel"
Thank you

Comment (andrescirulo) [reply src/dev/impl/DevToys/ViewModels/Tools/Generators/HashGenerator/HashGeneratorToolViewModel.cs:164]: I commited the changes, please check if now it's better

Comment (btiteux) [reply src/dev/impl/DevToys/ViewModels/Tools/Generators/HashGenerator/HashGeneratorToolViewModel.cs:164]: It's seems good to me @veler do you want to check too?

**Review by veler — APPROVED** [review]
Looking good, thanks for this contribution :)

**Pull request merged**